cmd/zc_traverser_local.go

```diff
@@ -104,12 +111,21 @@ func UnfurlSymlinks(symlinkPath string) (result string, err error) {
 
 			result = common.ToExtendedPath(result)
 
+			/*
+			 * Either we can store all the symlink seen till now for this path or we count how many iterations to find out cyclic loop.
+			 * Choose the count method and restrict the number of links to 40. Which linux kernel adhere.
+			 */
+			if count >= 40 {
```

Comment:
Getting in Adele's comments here. 40 is a magic number, should either derive this from syscall/os library if available. Else turn this a constant and comment appropriately.